website/static/events/seattle-summit-2025/index.html

```diff
@@ -0,0 +1,218 @@
+                            <svg width="24px" height="30px" viewBox="0 0 24 30" version="1.1" xmlns="http://www.w3.org/2000/svg"
+                                xmlns:xlink="http://www.w3.org/1999/xlink">
+                                <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+                                    <g id="location_on_FILL1_wght400_GRAD0_opsz48" fill="#000000" fill-rule="nonzero">
+                                        <path
+                                            d="M12,14.625 C12.725,14.625 13.34375,14.36875 13.85625,13.85625 C14.36875,13.34375 14.625,12.725 14.625,12 C14.625,11.275 14.36875,10.65625 13.85625,10.14375 C13.34375,9.63125 12.725,9.375 12,9.375 C11.275,9.375 10.65625,9.63125 10.14375,10.14375 C9.63125,10.65625 9.375,11.275 9.375,12 C9.375,12.725 9.63125,13.34375 10.14375,13.85625 C10.65625,14.36875 11.275,14.625 12,14.625 Z M12,30 C7.975,26.575 4.96875,23.39375 2.98125,20.45625 C0.99375,17.51875 0,14.8 0,12.3 C0,8.55 1.20625,5.5625 3.61875,3.3375 C6.03125,1.1125 8.825,0 12,0 C15.175,0 17.96875,1.1125 20.38125,3.3375 C22.79375,5.5625 24,8.55 24,12.3 C24,14.8 23.00625,17.51875 21.01875,20.45625 C19.03125,23.39375 16.025,26.575 12,30 Z"
+                                            id="Shape"></path>
+                                    </g>
+                                </g>
+                            </svg>
+                            Google SLU<br />
+                            620 Boren Ave N<br />
+                            Seattle, WA 98109
+                        </p>
+                    </div>
+                    <div class="grid-cell grid-cell--col6 grid-cell--col10--tablet">
+                        <p class="f32 f24--tablet paragraph-2">
+                            On October 8-9, 2025, come together with the
+                            <a href="https://vis.gl/" target="_blank" rel="noopener noreferrer">vis.gl</a>/<a
+                                href="https://deck.gl/" target="_blank" rel="noopener noreferrer">deck.gl</a>
+                            community to discuss the present & future of the leading open source
+                            library for visualization in the Web.
+                        </p>
+                    </div>
+                </div>
+        
+                <div class="grid-cell grid-cell--col10 u-mt80 gradient-border u-flex-column u-align-center">
+                    <p class="f32 f24--tablet paragraph-3">
+                        At the Open Visualization Collaborator Summit, we'll be bringing
+                        together an international audience of developers to discuss how
+                        they are using deck.gl to build apps for geospatial analysis, biological research, and AI training.
+                        The goal is to foster more contribution and
+                        envisage the future of the leading open source visualization library.
```

Comment:
```suggestion
                        At the Open Visualization Collaborator Summit, we’ll bring together an international audience of developers to discuss how they are using open-source tools like deck.gl, GeoDa, and Cosmos to build applications for geospatial analysis, biological research, AI training, and beyond.
Our goal is to foster collaboration, encourage contributions, and shape the future of open-source visualization across disciplines.
```

An idea on how to incorporate Cosmos and GeoDa